src/pages/background/userActions.js

```diff
@@ -66,16 +66,16 @@ export function contextClick(info, tab) {
 /* Saving
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
 async function save({ linkUrl, pageUrl, title, tabId }) {
-  // Are we authed?
-  const access_token = await getSetting('access_token')
-  if (!access_token) return logIn({ linkUrl, pageUrl, title, tabId })
-
   // send message that we are requesting a save
   chrome.tabs.sendMessage(tabId, { action: SAVE_TO_POCKET_REQUEST })
 
-  const url = linkUrl || pageUrl
-
   try {
+    // Are we authed?
+    const access_token = await getSetting('access_token')
+    if (!access_token) return logIn({ linkUrl, pageUrl, title, tabId })
+
+    const url = linkUrl || pageUrl
```

Comment:
These 3 lines came from above. The only difference here is that both checks to local storage are now wrapped with try, and we open the panel with the `SAVE_TO_POCKET_REQUEST` action immediately.

The visual difference is the panel will open with the "Saving..." text, and will remain until the user has completed the login process.